scripts/sct_pipeline_makefig.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python
+# -*- coding: utf-8
+#
+# Generate figure from the output of sct_pipeline.
+#
+# Copyright (c) 2013 Polytechnique Montreal <www.neuro.polymtl.ca>
+# Author: Julien Cohen-Adad
+# Created: 2018-05-21
+# License: https://github.com/neuropoly/spinalcordtoolbox/blob/master/LICENSE
+
+# TODO: generalize this code for other things than Dice
+
+import argparse
+
+
+def get_parser():
+    parser = argparse.ArgumentParser(
+        description='Generate figure from the output of sct_pipeline.')
+    parser.add_argument("-i",
```

Comment:
When using argparse for new code, suggest to use short/long arguments:
```
 "-i", "--input",
"-l", "--label",
"-v", "--verbose",
```

Reply:
fixed in 9d77c82139345e4600a708702d8ba06729c7a1f6